src/iexamine.cpp

```diff
@@ -653,7 +653,7 @@ void iexamine::cardreader(player &p, const tripoint &examp)
 
 void iexamine::rubble(player &p, const tripoint &examp)
 {
-    bool has_digging_tool = p.has_quality( "DIG", 2 );
+    bool has_digging_tool = p.has_quality( quality_id( "DIG" ), 2 );
```

Comment:
Out of curiosity why is `string_id<T>()` constructor marked `explicit`. Presumably this is why you require syntax such as this?